recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "20230802.1" %}
+{% set version = "20240116" %}
```

Comment:
```suggestion
{% set version = "20240116.0" %}
```

Reply:
Ugh, why does the version scheme change from rc to GA...